pep-0585.rst

```diff
@@ -112,6 +112,9 @@ Python 3.9, the following collections become generic using
 * ``collections.abc.ValuesView``
 * ``contextlib.AbstractContextManager``  # typing.ContextManager
 * ``contextlib.AbstractAsyncContextManager``  # typing.AsyncContextManager
+* ``re.Pattern``  # typing.Pattern, typing.re.Pattern
+* ``re.Match``  # typing.Match, typing.re.Match
+* ``io.IO``  # typing.IO, typing.io.IO
```

Comment:
Oh also we decided this isn't similar to `typing.IO`, right?

Reply:
Oh, yeah. I'll scratch this line. (Now I'm wondering if perhaps we shouldn't have messed with `IOBase` in the code branch?)

Reply:
Yeah I doubt people will have io.IOBase imported anyway, I'll make a PR to roll that back.

Reply:
OK, that makes sense. `io.IOBase` is not generic in typeshed. I just goofed up here.